modules/flowable-engine/src/test/java/org/flowable/examples/variables/VariablesTest.java

```diff
@@ -197,75 +206,77 @@ public void testLocalizeVariables() {
         ProcessInstance processInstance = runtimeService.startProcessInstanceByKey("localizeVariables", variables);
 
         Map<String, VariableInstance> variableInstances = runtimeService.getVariableInstances(processInstance.getId());
-        assertEquals(1, variableInstances.size());
-        assertEquals("stringVar", variableInstances.get("stringVar").getName());
-        assertEquals("coca-cola", variableInstances.get("stringVar").getValue());
+        assertThat(variableInstances.get("stringVar"))
+                .extracting("name", "value")
+                .containsExactly("stringVar", "coca-cola");
 
         List<String> variableNames = new ArrayList<>();
         variableNames.add("stringVar");
 
         // getVariablesInstances via names
         variableInstances = runtimeService.getVariableInstances(processInstance.getId(), variableNames);
-        assertEquals(1, variableInstances.size());
-        assertEquals("stringVar", variableInstances.get("stringVar").getName());
-        assertEquals("coca-cola", variableInstances.get("stringVar").getValue());
+        assertThat(variableInstances.get("stringVar"))
+                .extracting("name", "value")
```

Comment:
Let's not use reflection extraction of field names. Why not use `VariableInstance::getName, VariableInstance::getValue` instead? There are other places where this is done as well.

Reply:
Done